frontend/src/pages/Auth/VerifyEmail.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button } from '@components/Button'
+import {
+	AppLoadingState,
+	useAppLoadingContext,
+} from '@context/AppLoadingContext'
+import { useGetAdminQuery } from '@graph/hooks'
+import { Box, Stack, Text } from '@highlight-run/ui'
+import { AuthBody, AuthFooter, AuthHeader } from '@pages/Auth/Layout'
+import { Landing } from '@pages/Landing/Landing'
+import { auth } from '@util/auth'
+import { message } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import * as styles from './AuthRouter.css'
+
+export const VerifyEmail: React.FC = () => {
+	const { setLoadingState } = useAppLoadingContext()
+	const { data, stopPolling } = useGetAdminQuery({
+		pollInterval: 1000,
+	})
+	const [loading, setLoading] = useState(false)
+	const navigate = useNavigate()
+	const isEmailVerified = data?.admin?.email_verified || false
+
+	useEffect(() => {
+		if (isEmailVerified) {
+			stopPolling()
+			navigate('/about_you')
+		} else {
+			// Show the Intercom message after 5 seconds in case the user needs help.
+			setTimeout(() => {
+				window.Intercom('update', {
```

Comment:
Should we use the Intercom helper instead?
https://github.com/highlight/highlight/blob/main/frontend/src/util/window.ts#L17